### ISSUE TITLE
Time Recorded 

### ISSUE THREAD

**Issue by hobbitsyfeet** [hobbitsyfeet/PeopleTracker#10]

**Comment by hobbitsyfeet** [comment]
Total_rec_time(s) needs to become Total_rec_time(H,M,S). Each in their own column

Calculate time should try and subtract time if time segments move backwards. Include Read_only data, and maybe time_entered at Read_Only time. Also, look into simply removing segment limit if segments return negative...

**Comment by hobbitsyfeet** [comment]
It looks like start-end pairs to measure time do not join if ranges intersect.

For example, list([0,500], [300,600], [200,600]) can all exist. We want to join these ranges such that we get list([0,600])

**Comment by hobbitsyfeet** [comment]
Current Fix:
 - Sort input so that negative time does not exist. This eliminates countdown being "reverted" as you scroll backwards. This should be fine since reversing should be to adjust the location, not the time. BUT, if a person leaves and you continue to track, this cannot be undone. This will need a future fix.

- This aggregates all time that has been tracked through the whole video. Total Tracked Time will never be reduced, even when reversed. If new data is revisited and does not exist within the time interval, it will add it to Total Tracked Time. 

TLDR: Make sure you do not continue to track when a person leaves the frame, further fixes will need to be done to account for that. Other than that, Total Tracked Time is very accurate to the data captured, unlike before.
